src/node_sqlite.cc

```diff
@@ -1035,7 +1036,7 @@ bool StatementSync::BindValue(const Local<Value>& value, const int index) {
         statement_, index, *val, val.length(), SQLITE_TRANSIENT);
   } else if (value->IsNull()) {
     r = sqlite3_bind_null(statement_, index);
-  } else if (value->IsUint8Array()) {
+  } else if (value->IsArrayBufferView() || value->IsDataView()) {
```

Comment:
`ArrayBuffer.isView(new DataView(new ArrayBuffer))` is `true`, I think that second check is pure overhead
```suggestion
  } else if (value->IsArrayBufferView()) {
```